README.md

```diff
@@ -72,34 +74,44 @@ echo "your keystore password" > `pwd`/password.txt
 
 Then run the faucet application without the wallet command-line flags:
 ```bash
-./eth-faucet -httpport 8080
+./lsk-faucet -httpport 8080
 ```
 
 **Optional Flags**
 
 The following are the available command-line flags(excluding above wallet flags):
 
 | Flag              | Description                                      | Default Value |
-|-------------------|--------------------------------------------------|---------------|
+| ----------------- | ------------------------------------------------ | ------------- |
 | -httpport         | Listener port to serve HTTP connection           | 8080          |
 | -proxycount       | Count of reverse proxies in front of the server  | 0             |
-| -faucet.amount    | Number of Ethers to transfer per user request    | 1             |
+| -token-address    | Token contract address                           |               |
+| -faucet.amount    | Number of LSK to transfer per user request       | 1             |
 | -faucet.minutes   | Number of minutes to wait between funding rounds | 1440          |
-| -faucet.name      | Network name to display on the frontend          | testnet       |
-| -faucet.symbol    | Token symbol to display on the frontend          | ETH           |
+| -faucet.name      | Network name to display on the frontend          | sepolia       |
+| -faucet.symbol    | Token symbol to display on the frontend          | LSK           |
 | -hcaptcha.sitekey | hCaptcha sitekey                                 |               |
 | -hcaptcha.secret  | hCaptcha secret                                  |               |
 
 ### Docker deployment
+#### Build docker image
+Run the following command to build docker image:
+```bash
+docker build -t liskhq/lsk-faucet .
+```
+
+
+#### Run faucet
+Run the following command to start the application:
 
 ```bash
-docker run -d -p 8080:8080 -e WEB3_PROVIDER=rpc endpoint -e PRIVATE_KEY=hex private key chainflag/eth-faucet:1.1.0
+docker run -d -p 8080:8080 -e WEB3_PROVIDER=rpc endpoint -e PRIVATE_KEY=hex private key liskhq/lsk-faucet
 ```
```

Comment:
````suggestion
docker run -d -p 8080:8080 -e WEB3_PROVIDER=<rpc-endpoint> -e PRIVATE_KEY=<hex-private-key> liskhq/lsk-faucet
```
**NOTE**: Please replace `<rpc-endpoint>` and `<hex-private-key>` with appropriate values.
````